internal/kibana/slo.go

```diff
@@ -541,6 +542,17 @@ func getSloFromResourceData(d *schema.ResourceData) (models.Slo, diag.Diagnostic
 		}
 
 	case "metric_custom_indicator":
+		goodMetricsRaw := d.Get(indicatorType + ".0.good.0.metrics").([]interface{})
```

Comment:
It seems that in the SLO UI the `total` can also accept an array of metrics. Should we also support an array of metrics for it?

<img width="809" alt="Screenshot 2024-01-29 at 11 11 02 AM" src="https://github.com/elastic/terraform-provider-elasticstack/assets/7871006/35bd330d-4ea8-4a3b-970b-df15ead097d1">